pkg/vault/contracts/VaultAdmin.sol

```diff
@@ -308,6 +308,8 @@ contract VaultAdmin is IVaultAdmin, VaultCommon, Authentication, VaultGuard {
         onlyProtocolFeeController
     {
         _poolConfigBits[pool] = _poolConfigBits[pool].setAggregateSwapFeePercentage(newAggregateSwapFeePercentage);
+
+        emit AggregateSwapFeePercentageChanged(pool, newAggregateSwapFeePercentage);
```

Comment:
Not completely related to this PR, but the vault doesn't validate the fee precision (only the protocol fee controller does).

Should we validate it here? If the protocol fee controller doesn't validate it well, the value that gets set in storage won't really match the emitted event. Super minor, but still a possibility I guess?